core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -162,6 +180,14 @@
           </li>
           <li></li>
         </ul>
+        <ul class="mobile-feedback" ng-if="windowIsNarrow">
+          <li popover-placement="top" popover-template="'popover/feedback'" popover-trigger="click">
+            <a href="" class="btn oppia-navbar-button" tooltip="<['I18N_PLAYER_FEEDBACK_TOOLTIP' | translate]>" tooltip-placement="top">
+              <i class="material-icons">&#xE87F;</i>
+              <span><['I18N_PLAYER_FEEDBACK_TOOLTIP' | translate]></span>
```

Comment:
Could we go with `span translate="..."` here? (We generally do it like this unless there's no alternative.)